### PR TITLE
Create an ES6 import program

### LINKED ISSUE #101: Create an ES6 import program
Right now we have two mock programs under the `test/` directory, `bad-structure` and `pizza-ordering`. These programs are designed to resemble a real program built on Waterfall CLI.

We should add one additional program, called `es6-imports`, with the following structure:

```
test/
   programs/
      es6-imports/
         cli/
            entry.js
            entry.json
         package.json
```

The `entry.json` file just needs to be empty: `{}`. It doesn't need any special options or flags.

Then, we need to add a `test/es6-imports.test.js` file that resembles the `pizza-ordering.test.js` file, except that it:

- Asks the `runTest()` function to run Node.js with the `--experimental-modules` flag.
- Only runs one test; specifically, the `Displays version` test.

### PR DESCRIPTION
**Pull request by pbelbin**
Closes #101

### CODE REVIEW

**Comment by pbelbin** [comment]
Strangely, when the `npm test` command is used, the test completes just fine.

But: when invoked manually from the command line, it does not work.
Image 2019-10-03 at 1 26 59 PM

File: test/es6-imports.test.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/* eslint-env jest */
+
+// Dependencies
+const runProgram = require('./run-program');
+
+// Initialize
+const entryFile = `${__dirname}/programs/pizza-ordering/cli/entry.js`;
+
+// Tests
+describe('Pizza ordering', () => {
```
Comment (WesCossick) [inline test/es6-imports.test.js]: This needs to be

```suggestion
describe('ES6 imports', () => {
```

File: test/es6-imports.test.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/* eslint-env jest */
+
+// Dependencies
+const runProgram = require('./run-program');
+
+// Initialize
+const entryFile = `${__dirname}/programs/pizza-ordering/cli/entry.js`;
```
Comment (WesCossick) [inline test/es6-imports.test.js]: You're not referencing the correct entry point:

```suggestion
const entryFile = `${__dirname}/programs/es6-imports/cli/entry.js`;
```

File: test/es6-imports.test.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+// Tests
+describe('Pizza ordering', () => {
+	describe('Built-in abilities', () => {
+		test('Displays version', () => {
+			return runProgram(entryFile, '--version', '--experimental-modules').then(
+				({ stdout }) => {
+					expect(stdout.includes('pizza-ordering: 1.2.3')).toBe(true);
```
Comment (WesCossick) [inline test/es6-imports.test.js]:
```suggestion
					expect(stdout.includes('es6-imports: 1.2.3')).toBe(true);
```

**Pull request merged**